hack/deploy-logging.sh

```diff
@@ -177,6 +177,7 @@ EXTERNAL_REGISTRY=${EXTERNAL_REGISTRY:-registry.svc.ci.openshift.org}
 EXT_REG_IMAGE_NS=${EXT_REG_IMAGE_NS:-origin}
 # for dev purposes, image builds will typically be pushed to this namespace
 OPENSHIFT_BUILD_NAMESPACE=${OPENSHIFT_BUILD_NAMESPACE:-openshift}
+IMAGESTREAM_NAME=${IMAGESTREAM_NAME:-$MASTER_VERSION}
```

Comment:
since this is `feature-es6x` can't we just hardcode this for now?

Reply:
> since this is `feature-es6x` can't we just hardcode this for now?

We could but I'm expecting the changes in https://github.com/openshift/release/pull/6843 to make so we dont have to modify it again once we merge it back into master.  This change is likely what is need in order for the rehearse to succeed